apartment/models.py

```diff
@@ -0,0 +1,72 @@
+from django.db import models
+
+from core      import models as core_models
+
+class Apartment(core_models.TimeStampModel):
+    price             = models.DecimalField(max_digits=20, decimal_places=4)
+    top_floor         = models.IntegerField()
+    floor             = models.IntegerField()
+    trade_date        = models.DateTimeField(auto_now=True, null=True)
+    apartment_complex = models.ForeignKey('ApartmentComplex', on_delete=models.CASCADE)
+    district          = models.ForeignKey('District', on_delete=models.SET_NULL, null=True)
+    trade_type        = models.ForeignKey('TradeType', on_delete=models.SET_NULL, null=True)
+    size              = models.ManyToManyField('Size', through='ApartmentSize')
+
+    class Meta:
+        db_table = 'apartments'
+
+class ApartmentComplex(models.Model):
+    name             = models.CharField(max_length=100)
+    household_number = models.IntegerField()
+    completion_year  = models.CharField(max_length=20)
+    image_url        = models.URLField(max_length=1000, null=True)
+    address          = models.CharField(max_length=1000)
+    longitude        = models.DecimalField(max_digits=30, decimal_places=25)
```

Comment:
좌표값 치고는 범위가 좀 큰것 같아요 
```suggestion
    longitude        = models.DecimalField(max_digits=20, decimal_places=15)
```
정도로 조절해 주세요.